### ISSUE TITLE
osx support

### ISSUE THREAD

**Issue by tofutim** [sdcb/PaddleSharp#110]
### Feature request type

sample request

### Is your feature request related to a problem? Please describe

What prevents PaddleSharp from working on Mac OSX (Intel or Apple Silicon)? Is it a hard problem?

### Describe the solution you'd like

It works!

**Comment by n0099** [comment]
The underlaying PaddlePaddle isn't supporting it.

**Comment by grinay** [comment]
@n0099 this is not true. I'm working on product where we use paddlepaddle. And I' using it on my osx m3. The only thing it works only on CPU, and I had to build it on my own. You can use my package PaddleInference.runtime.osx-arm64 along with paddlesharp.

**Comment by kwasak** [comment]
> [@n0099](https://github.com/n0099) this is not true.@ [@tofutim](https://github.com/tofutim) I'm working on product where we use paddlepaddle. And I' using it on my osx m3. The only thing it works only on CPU, and I had to build it on my own. You can use my package PaddleInference.runtime.osx-arm64 along with paddlesharp.

Hey. I'm using your package PaddleInference.runtime.osx-arm64 and can't seem to get it to work. I'm getting segmentation fault (Code 139), when using Mkldnn or Openblas. MLKDNN also says "Please compile with MKLDNN first to use MKLDNN". What device did you use? M1 Mac.

**Comment by grinay** [comment]
@kwasak hi. Yes, I’m using it on two different apple silicon devices m1 and m3. 
Let’s try to understand the issue. 
What .net version are you using. 
Also let’s print some debugging 
Set this env variable for running your application it will give you insights about what libs actually loaded. 
export DYLD_PRINT_LIBRARIES=1
export DYLD_PRINT_LIBRARIES_POST_LAUNCH=1
export DYLD_PRINT_RPATHS=1

Or run in console something like this DYLD_PRINT_LIBRARIES=1 dotnet yourapp.dll

**Comment by sdcb** [comment]
Hi, osx support is ready and now in CICD
